broker/src/main/java/io/zeebe/broker/system/monitoring/BrokerHealthCheckService.java

```diff
@@ -83,4 +105,28 @@ private void initializePartitionInstallStatus() {
   public String getName() {
     return actorName;
   }
+
+  @Override
+  protected void onActorStarted() {
+    healthMonitor.startMonitoring();
+  }
+
+  public void registerComponent(final String componentName, final HealthMonitorable component) {
+    actor.call(
+        () -> {
```

Comment:
Let's use the expression lambda syntax where possible (e.g. if a single line, no need to use brackets `{`)